lib/bootscale/entry.rb

```diff
@@ -13,6 +13,7 @@ def initialize(path)
     end
 
     def features
+      return [] unless @path.start_with?('/'.freeze)
```

Comment:
How would that change behavior ?
You'll get cache misses -> normal require takes over

On Tue, Aug 25, 2015 at 8:51 AM, Jean Boussier <notifications@github.com>
wrote:

> In lib/bootscale/entry.rb
> <https://github.com/byroot/bootscale/pull/18#discussion_r37882159>:
>
> > @@ -13,6 +13,7 @@ def initialize(path)
> >      end
> >
> >      def features
> > +      return [] unless @path.start_with?('/'.freeze)
>
> I think we should blow up. Because this could potentially change the
> behavior.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/byroot/bootscale/pull/18/files#r37882159>.
>